daemon/api_snaps.go

```diff
@@ -56,10 +56,11 @@ var (
 	}
 
 	snapsCmd = &Command{
-		Path:        "/v2/snaps",
-		GET:         getSnapsInfo,
-		POST:        postSnaps,
-		ReadAccess:  openAccess{},
+		Path: "/v2/snaps",
+		GET:  getSnapsInfo,
+		POST: postSnaps,
+		// TODO: maybe only allow `select=refresh-inhibit`?
+		ReadAccess:  interfaceOpenAccess{Interface: "snap-refresh-inhibit-observe"},
```

Comment:
would it be useful to allow `/v2/snaps/{name}` too?

Reply:
I don't mind adding it, but it is not directly needed by in flow described in the spec [SD167](https://docs.google.com/document/d/1HJQWKzgaB3yPKwRUqlxYhgyaKG5IcJe8eZOT4YY1CI8/edit) but maybe it could be useful in some cases not covered by the spec. @pedronis would love to know your opinion here.